main.tf

```diff
@@ -35,6 +47,28 @@ resource "aws_cloudfront_distribution" "api_gate" {
     max_ttl                = 31536000
   }
 
+  ordered_cache_behavior {
+    path_pattern    = "/middleware/*"
+    allowed_methods = ["HEAD", "DELETE", "POST", "GET", "OPTIONS", "PUT", "PATCH"]
+    cached_methods  = ["GET", "HEAD"]
+
+    forwarded_values {
+      query_string = true
+
+      cookies {
```

Comment:
We don't need cookies forward for this origin, AFAIK

Reply:
0c1083a